README.md

```diff
@@ -143,8 +41,12 @@ The desktop app uses the [Electron](https://www.electronjs.org/) framework. It p
 being different account storage.
 
 The accounts are stored in the following directories:
-* Mac: `~/Library/Application Support/polkadot-apps/polkadot-accounts`
-* Linux: `~/.config/polkadot-apps/polkadot-accounts` (or `$XDG_CONFIG_HOME/polkadot-apps/polkadot-accounts` if `$XDG_CONFIG_HOME` is defined)
-* Windows: `%APPDATA%\polkadot-apps\polkadot-accounts`
+* Mac: `~/Library/Application Support/azero-dev/polkadot-accounts`
+* Linux: `~/.config/azero-dev/polkadot-accounts` (or `$XDG_CONFIG_HOME/azero-dev/polkadot-accounts` if `$XDG_CONFIG_HOME` is defined)
+* Windows: `%APPDATA%\azero-dev\polkadot-accounts`
 
 For more details on the desktop app, head over to [Electron package README](https://github.com/polkadot-js/apps/blob/master/packages/apps-electron/README.md).
+
+## Credits
+
+This repos is a fork of https://github.com/polkadot-js/apps.
```

Comment:
After all the renamings, I thought it's a right thing to do to leave a clear not that this repo is heavily inspired by the Polkadot's one.